metadata/bootstrap_nodes.yaml

```diff
@@ -0,0 +1,13 @@
+# holesky consensus layer bootnodes
+# ---------------------------------------
+# 1. Tag nodes with maintainer
+# 2. Keep nodes updated
+# 3. Review PRs: check ENR duplicates, fork-digest, connection.
+
+# EF
+- enr:-Ku4QPf_feGn3gFpYlUhiuEMH2_RLhLm2LpYM00z8ygFYIbkJyQuK6M75yXATVi_sn53X30a9E_W9DhYzfwkPEPTRm0Ch2F0dG5ldHOIAAAAAAAAAACEZXRoMpDe38V2EAAJEP__________gmlkgnY0gmlwhIbHuBeJc2VjcDI1NmsxoQP3FwrhFYB60djwRjAoOjttq6du94DtkQuaN99wvgqaIYN1ZHCCIyk
```

Comment:
note this would still need to be updated with the new genesis files once confirmed